pegasus/sites.v3/code.org/public/curriculum/text-compression/1/Teacher.moved

```diff
@@ -1 +1 @@
-https://curriculum.code.org/hoc/2016/1/
\ No newline at end of file
+https://curriculum.code.org/hoc/unplugged/2/
```

Comment:
this file was different in that it existed already, but was pointing to the wrong location. however, the old lesson plan was not previously accessible, because has_lesson_plan was false, so the View Lesson Plan button was not visible.